### PR TITLE
[Sprint #5]Signup FormView & Sending a mail using Celery

### PR DESCRIPTION
**Pull request by jinwookim0**

### CODE REVIEW

**Comment by jinwookim0** [comment]
The result of Travis CI build is because of our password policy. Our mail password is loaded to our env variable path.

**Comment by jinwookim0** [comment]
I just passed travis test now. There was a problem. because my previous code pull local environment variable, so travis detected that line, and regulated this is a bug(failure). so I added an export instruction about our password environment  information in .travis.yml. (also Encrypted. [secure]). Then, it worked.

**Comment by dobestan** [comment]
Congratulation :)

File: buildbuild/users/tests/test_signup_page.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    def test_post_signup_page_with_available_user_information_should_redirect_to_home(self):
+        response = self.client.post("/signup/", {
+            "email": self.valid_email,
+            "password": self.valid_password,
+            })
+        self.assertEqual(response["Location"], self.TEST_SERVER_URL + "/")
+   
+    
+    #needed to change the codes... Because create_user raises an error in case of invalid password
```
Comment (dobestan) [inline buildbuild/users/tests/test_signup_page.py]: which test case needs to change ?

File: buildbuild/users/tests/test_send_mail.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+from django.db.utils import OperationalError
+
+from users import tasks
+
+class SignUpPageTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.valid_email = "test@example.com"
+        self.valid_password = "test_password"
+        self.invalid_password = "a"*5
```
Comment (dobestan) [inline buildbuild/users/tests/test_send_mail.py:19]: in this test cases ( `test_send_mail.py` ) there is no test cases with `self.invalid_password`, and why defined ?

File: buildbuild/users/tests/test_signup_page.py (hunk 1 above)
Comment (jinwookim0) [inline buildbuild/users/tests/test_signup_page.py]: ah.. I knew that comment. that problem commented is solved. some corresponding exception was added. (IntegrityError, and OperationalError). I will remove that comment now.

File: buildbuild/users/tests/test_send_mail.py (hunk 2 above)
Comment (jinwookim0) [inline buildbuild/users/tests/test_send_mail.py:19]: Oh, thanks, Okay, I will add test code about authentication more detail

**Pull request merged**